Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
Ok.  Since this is not using gulp to begin with, I think I will stick with Make here to hopefully keep it consistent with other repos.  For the jshintrc file,  does it make sense to copy the one from the UI toolkit here, or somehow install it?